lib/src/rules/overriden_field.dart

```diff
@@ -79,8 +97,21 @@ class _Visitor extends SimpleAstVisitor {
   @override
   visitFieldDeclaration(FieldDeclaration node) {
     node.fields.variables.forEach((VariableDeclaration variable) {
-      ExecutableElement member = _getOverriddenMember(variable.element);
-      if (member is PropertyAccessorElement && member.isSynthetic) {
+      PropertyAccessorElement field = _getOverriddenMember(variable.element);
+      ClassElement variableClassElement = variable.element.enclosingElement;
```

Comment:
Given that most of the time the field should be `null`, the code could be more efficient by enclosing everything from here down in an `if (field != null)` statement. Otherwise the code will create the list of implemented types once for every field.